client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -72,11 +72,13 @@ const DataFilesCopyModal = React.memo(() => {
   const copyCallback = useCallback(
     (system, path) => {
       setDisabled(true);
-      const filteredSelected = selected.filter(f => status[f.id] !== 'SUCCESS');
+      const filteredSelected = selected
```

Comment:
I know from experience google breadcrumbs are a nightmare:
When navigating into a subfolder, breadcrumbs show the file id (path) instead of the name. The breadcrumbs
<img width="487" alt="Screen Shot 2020-12-17 at 12 40 01 AM" src="https://user-images.githubusercontent.com/20326896/102452792-95a92500-4000-11eb-94b1-22ccb3591a00.png">
 also only ever show a depth of one
<img width="417" alt="Screen Shot 2020-12-17 at 12 31 48 AM" src="https://user-images.githubusercontent.com/20326896/102452687-61356900-4000-11eb-815c-5d40bf66702e.png">

Reply:
I vote we come back to this. I think the solution ultimately is going to be to truncate the breadcrumbs to /Google Drive/.../${folderName} no matter where you are, since google drive doesn't actually have a model for file paths. That's going to involve returning an additional value from the file listing, which I'm hesitant to do while so much of Data Files is in flux.

Reply:
Sounds good to me. I could have sworn we figured this out in DesignSafe, but I could be wrong